src/proto/graplinc/grapl/api/graph_query_service/v1beta1/graph_query_service.proto

```diff
@@ -0,0 +1,340 @@
+syntax = "proto3";
+
+package graplinc.grapl.api.graph_query_service.v1beta1;
+
+import "graplinc/common/v1beta1/types.proto";
+import "graplinc/grapl/common/v1beta1/types.proto";
+
+// A filter for querying an integer
+message IntFilter {
+  // The operation to filter with
+  enum Operation {
+    // An unknown operation
+    OPERATION_UNSPECIFIED = 0;
```

Comment:
Is the `OPERATION_` prefix really needed on all these? I'd think the fact that they're all part of an `Operation` enum would be sufficient.

Reply:
Actually, yes! Our linter enforces this. It's because some languages don't do namespacing well iirc
![Screenshot 2022-08-16 10 47 19 AM](https://user-images.githubusercontent.com/69007229/184909650-f92a15e0-941b-4817-875f-e825df99337a.png)

Reply:
Yeah, there's no choice here - tonic helpfully removes the redundancy in generated code though.